# Parameters for the loop and lemniscate trajectories at successively increasing speed
loop_z: 2.5              # Z position of loop plane [m]
loop_r: 5.0            # Radius of loop [m]
loop_v_max: 10.0       # Maximum speed achieved (approx.) [m/s]
loop_lin_a: 0.25       # Linear acceleration and deceleration of trajectory [m/s^2]
loop_clockwise: False  # Rotation direction of the quad
loop_yawing: False     # Yaw quadrotor along the circle (only for loop)